[autolinking] Introduce patcher for double-quoted import issue (#15655)
expo/expo: packages/expo-modules-autolinking/CHANGELOG.md

### 🐛 Bug fixes

> - Introduce `ReactImportsPatcher` to transform double-quoted react imports into angle-brackets and fix incompatible build errors from third-party libraries on SDK 44. ([#15655](https://github.com/expo/expo/pull/15655) by [@kudo](https://github.com/kudo))

Reviewer: In changelogs we should use publicly exposed names over internal ones (if possible).
```suggestion
- Introduce `expo_patch_react_imports!` to transform double-quoted React imports into angle-brackets in order to fix third-party libraries incompatibility with SDK 44. ([#15655](https://github.com/expo/expo/pull/15655) by [@kudo](https://github.com/kudo))
```
